### PR TITLE
MRG Fix example.

### PR DESCRIPTION
**Pull request by jaeilepp**

### CODE REVIEW

**Comment by Eric89GXL** [comment]
LGTM, +1 for merge

**Comment by jaeilepp** [comment]
Except the example I was trying to fix still does not work. https://1019-35097627-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/decoding/plot_decoding_spatio_temporal_source.html#sphx-glr-auto-examples-decoding-plot-decoding-spatio-temporal-source-py

**Comment by Eric89GXL** [comment]
We need a "days since last Mayavi CI problem" counter

File: examples/inverse/plot_lcmv_beamformer.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
 event_id, tmin, tmax = 1, -0.2, 0.5
 
 # Setup for reading the raw data
-raw = mne.io.read_raw_fif(raw_fname, preload=True, proj=True)
```
Comment (agramfort) [inline examples/inverse/plot_lcmv_beamformer.py]: does it affect the results? as it's not applied below on Epochs

**Comment by jaeilepp** [comment]
All good now. For some reason views as a list does not work on circle. It works fine for me locally.

File: examples/inverse/plot_lcmv_beamformer.py (hunk 1 above)
Comment (jaeilepp) [inline examples/inverse/plot_lcmv_beamformer.py]: Now it is applied.

**Comment by Eric89GXL** [comment]
There is a hack in PySurfer to get it to render in multiple view mode, it
can fail

**Comment by Eric89GXL** [comment]
Examples LGTM, +1 for merge

**Comment by agramfort** [comment]
if result of example is unchanged compared to master +1 for merge

**Comment by Eric89GXL** [comment]
@jaeilepp feel free to double-check the outputs compared to `master` and merge then

**Comment by jaeilepp** [comment]
It has actually been broken for a while on master. This returns it to the 'old' state from a while back. https://2470-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/auto_examples/decoding/plot_decoding_spatio_temporal_source.html#sphx-glr-auto-examples-decoding-plot-decoding-spatio-temporal-source-py

**Comment by jaeilepp** [comment]
I'll  backport this to  maint/0.13

**Pull request merged**